src/routes/chat/components/messages/Messages.js

```diff
@@ -5,21 +5,39 @@ import isEmpty from 'lodash/isEmpty';
 
 import Message from '../message';
 
-const testValueIsMe = true;
+// const testValueIsMe = true;
+const testUserId = 1;
 
 function Messages({ items }) {
   console.info('In messages', items);
   return (
     <div>
       {!isEmpty(items) &&
         map(items, item => {
+          let isMe = false;
```

Comment:
Could you move this logic into Message component somehow?

Reply:
I`m will think about it

Reply:
Do please, because you have too complex logic inside render, even some functions